src/menu.c

```diff
@@ -2780,11 +2776,11 @@ static  void    cb_disk_attr_clicked(UNUSED_WIDGET, void *p)
 
     dialog_destroy();
 
-    switch ((int)p) {
-    case DATA_DISK_ATTR_RENAME:  sub_disk_attr_rename(name);    break;
-    case DATA_DISK_ATTR_PROTECT: sub_disk_attr_protect(name);   break;
-    case DATA_DISK_ATTR_FORMAT:  sub_disk_attr_format(name);    break;
-    case DATA_DISK_ATTR_BLANK:   sub_disk_attr_blank();     break;
+    switch ((intptr_t)p) {
+    case DATA_DISK_ATTR_RENAME:   sub_disk_attr_rename(name);   break;
```

Comment:
```suggestion
    case DATA_DISK_ATTR_RENAME:  sub_disk_attr_rename(name);   break;
```
Extra space is unnecessary.